tests/e2e-inst-test.sh

```diff
@@ -13,7 +13,7 @@ SCRIPT_TMP_DIR=/tmp
 TRACEE_TMP_DIR=/tmp/tracee
 
 # Default test to run if no other is given
-TESTS=${INSTTESTS:=VFS_WRITE}
+TESTS=${INSTTESTS:=WRITABLE_DATA_SOURCE}
```

Comment:
@rafaeldtinoco I forgot to revert this, if you want to merge as is and can't wait for me to fix, go ahead and edit this out.